src/sentry/api/endpoints/organization_plugins.py

```diff
@@ -6,21 +6,41 @@
 from sentry.api.bases.organization import OrganizationEndpoint
 from sentry.api.serializers import serialize
 from sentry.api.serializers.models.organization_plugin import OrganizationPluginSerializer
-from sentry.models import Project
+from sentry.models import Project, ProjectOption
 
 
 class OrganizationPluginsEndpoint(OrganizationEndpoint):
     def get(self, request, organization):
-        _plugins = []
-
-        for project in Project.objects.filter(organization=organization):
-            for plugin in plugins.configurable_for_project(project, version=None):
-                _plugins.append(
-                    serialize(
-                        plugin,
-                        request.user,
-                        OrganizationPluginSerializer(project),
-                    )
+        # Just load all Plugins once.
+        all_plugins = dict([
+            (p.slug, p) for p in plugins.all()
+        ])
+
+        if 'plugins' in request.GET:
+            desired_plugins = request.GET['plugins'].split()
```

Comment:
We should validate this input. Otherwise this is unsanitized I out being plowed directly into ProjectOption keys.

Reply:
Doesn't Django (or the DB driver) take care of escaping things automatically? Either way, I'll add some validation.